Gem/Swap/Views/PriceImpactView.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+import Foundation
+import SwiftUI
+import Primitives
+import Components
+import Style
+
+struct PriceImpactView: View {
+    let fromAsset: AssetData
+    let fromValue: String
+    let toAsset: AssetData
+    let toValue: String
+    
+    let model: PriceImpactViewModel
+    
+    var body: some View {
+        let priceImpactValue = model.priceImpactValue(fromValue: fromValue, toValue: toValue)
+        
+        switch priceImpactValue {
+        case .none, .low:
+            EmptyView()
+        case .medium(let value):
```

Comment:
```suggestion
        case .medium(let value), high(let value), .positive(let value):
```

simplify it, because only `color` is different. create var color based on value